resources/profiles/Qidi/machine/Qidi X-Max 0.4 nozzle.json

```diff
@@ -100,7 +100,7 @@
 	"default_filament_profile": [
 		"Qidi Generic PLA"
 	],
-	"machine_start_gcode": "G28\nG92 E0\nG0 X5 Y5 Z0.3 F3600\n",
+	"machine_start_gcode": "G28\nG92 E0\nG0 X5 Y5 Z0.3 F3600\nM190 S[bed_temperature_initial_layer_single]\nM109 S[first_layer_temperature]\n",
```

Comment:
Sure ill add the default Qidi code for it

Reply:
@SoftFever 
This will make the bed raise during the warm-up phase.
Code is a copy from the Qidi slicer for both X-Plus and X-Max (excluding the first pre-print line)

X-Max Qidi code:
```
M82 ;absolute extrusion mode
G28
G0 X300 Y5 Z50 F3600
M190 S63
M109 S215.0
G92 E-19
G0 Y5 Z0.3 F3600
G1 X5 E0 F2400
```

Reply:
`G92 E-19`
Is this right? Feels it's retracting too much that will affect the normal printing

Reply:
Maybe @HYzd766 can shed some light here as well

Reply:
> `G92 E-19` Is this right? Feels it's retracting too much that will affect the normal printing Just curious as I have no experience on these two machines.

I'm just taking the vendor start g-code. I use the X-Plus values currently with Orca with good results.
Here are screenshots of the code generated with Qidi Print default profile (0.4 nozzle) for the printer.
X-Max:
![image](https://github.com/SoftFever/OrcaSlicer/assets/21292774/5367337c-be14-4d3f-a7e8-fffdda2843be)
X-Plus:
![image](https://github.com/SoftFever/OrcaSlicer/assets/21292774/ff419b19-5d5a-4270-a9d5-7b7bd8cf5a5a)

I can remove the retract but why mess with the vendor recommended values?

Reply:
> > `G92 E-19` Is this right? Feels it's retracting too much that will affect the normal printing Just curious as I have no experience on these two machines.
> 
> I'm just taking the vendor start g-code. I use the X-Plus values currently with Orca with good results. Here are screenshots of the code generated with Qidi Print default profile (0.4 nozzle) for the printer. (Qidi Print is a fork of Cura) X-Max: ![image](https://private-user-images.githubusercontent.com/21292774/324880908-5367337c-be14-4d3f-a7e8-fffdda2843be.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0MDU4NDEsIm5iZiI6MTcxNDQwNTU0MSwicGF0aCI6Ii8yMTI5Mjc3NC8zMjQ4ODA5MDgtNTM2NzMzN2MtYmUxNC00ZDNmLWE3ZTgtZmZmZGRhMjg0M2JlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDE1NDU0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2ZGQ1Zjg3MTRhYzkyNzdmMTBkMWNkZTk5NmQ3NTllMmM0ZmQ4MDZhNTkxMDVmMTZmYjliMjBhMzRkZDY2YjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rgVm2vU_1l9NeoHfiHriCqs2udjxIaeKEojcq_wVUqk) X-Plus: ![image](https://private-user-images.githubusercontent.com/21292774/324881061-ff419b19-5d5a-4270-a9d5-7b7bd8cf5a5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0MDU4NDEsIm5iZiI6MTcxNDQwNTU0MSwicGF0aCI6Ii8yMTI5Mjc3NC8zMjQ4ODEwNjEtZmY0MTliMTktNWQ1YS00MjcwLWE5ZDUtN2I3YmQ4Y2Y1YTVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDE1NDU0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY2NjE0Yjc0MmYxMDE2Y2RkZDYyZGY5Y2UyMDEzZDE5NTQwZGE1ZjdhNmIzMTE2NTc0YTE0YWNlNzJmN2ExZjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.321zWhYcSdEsMZvjcxyYSuyKBhXF9KOl4KqST0Se02I)
> 
> I can remove the retract but why mess with the vendor recommended values?

Sorry for the late response.
Just to confirm, you tested with these settings(include the  `G92 E-19` line in start gcode) on your machine, right?
As long as it's working on your side, I'm fine to take it.

Reply:
Hello @SoftFever 
NP :-) I have the X-plus printer and default for that printer is "G92 E-16" as in the pull request. 
G92 E-16 works good for X-plus. 

E-19 is default for Qidi X-max. 

I just copied what the vendor Qidi Slicer have. I can't see why it would fail on X-max as it's default and near. 

BTW without the changes both X-max and X-plus are non functional.

Reply:
> Hello @SoftFever NP :-) I have the X-plus printer and default for that printer is "G92 E-16" as in the pull request. G92 E-16 works good for X-plus.
> 
> E-19 is default for Qidi X-max.
> 
> I just copied what the vendor Qidi Slicer have. I can't see why it would fail on X-max as it's default and near.
> 
> BTW without the changes both X-max and X-plus are non functional.

Noted.
Thanks for the confirmation!